core/src/main/scala/kafka/zk/ZkData.scala

```diff
@@ -1103,11 +1105,11 @@ object ZkData {
     IsrChangeNotificationZNode.path,
     ProducerIdBlockZNode.path,
     LogDirEventNotificationZNode.path
-  ) ++ ConfigType.all.map(ConfigEntityTypeZNode.path)
+  ) ++ JavaConverters.asScalaIteratorConverter(ConfigType.ALL.iterator()).asScala.map(ConfigEntityTypeZNode.path)
```

Comment:
Same as above, can we build this collection without using `JavaConverters`?

Reply:
Great catch. Thanks. Rewrote like you suggested.